src/Illuminate/Routing/Router.php

```diff
@@ -848,6 +845,23 @@ public function middleware($name, $class)
     }
 
     /**
+     * Bulk register short-hand names for a set of middleware.
+     *
+     * Middlewares are passed through with the format [ 'name' => 'class', ... ]
+     *
+     * @param array $middlewares
```

Comment:
Give these a bit more space ;) There should be two spaces between them `@param  array  $middlewares`